src/uct/tcp/tcp_sockcm.c

```diff
@@ -175,6 +175,14 @@ UCS_CLASS_INIT_FUNC(uct_tcp_sockcm_t, uct_component_h component,
                               &uct_tcp_sockcm_iface_ops, worker, component,
                               config);
 
+    if (cm_config->priv_data_len < sizeof(uct_tcp_sockcm_priv_data_hdr_t)) {
+        ucs_error("provided CM private data length (%" PRIu64 ") is too small,"
```

Comment:
priv_data_len should configure how much provate data is there **in addition to** sizeof(uct_tcp_sockcm_priv_data_hdr_t)